Makefile

```diff
@@ -17,7 +17,24 @@ create_database: .env
 	docker compose run server create_db
 
 clean:
-	docker compose down && docker compose rm
+	docker compose down
+	docker compose --project-name cypress down
+	docker compose rm --stop --force
+	docker compose --project-name cypress rm --stop --force
+	docker compose rm
```

Comment:
This is looking more complete, but I think we do not need `docker compose rm` since this is already done by `docker compose rm --stop --force`.

Reply:
Heh Heh Heh.  Ok, I'll take another pass at it... :wink:

Reply:
Looking at it now, yeah you're right.  Not sure how I doubled that up.  Must have been really tired or something. :smile:

Reply:
k, I've just removed that doubling up and verified it's still all working as intended.  It cleanly removes the running redash and cypress containers, and removes the corresponding image files as well.

Reply:
Hmmm, I'll rebase this on `master` too while I'm at it...